nixos/modules/services/logging/logstash.nix

```diff
@@ -88,7 +88,7 @@ in
         example = ''
           # Read from journal
           pipe {
-            command => "''${pkgs.systemd}/bin/journalctl -f -o json"
+            command => "${pkgs.systemd}/bin/journalctl -f -o json"
```

Comment:
You are right. '' before $ cause it not to execute the expression? I think it's just confusing for those unfamiliar with the language but that shouldn't matter.